f76/scripts/scrape/junk_locations.py

```diff
@@ -1,32 +1,159 @@
 
+import pathlib
+from bs4 import BeautifulSoup, Tag
 
-# Now I want to write a script that's a bit different
+from .infra import db_conn, fetch_soup
+from ..parsing_utils import clean_text
 
-"""
-We know that each junk item has a URL associated with it:
+BASE = "https://fallout.fandom.com"
 
-.44 Casing - URL: https://fallout.fandom.com/wiki/.44_casing
+_NUM_WORDS = {
+    "one": 1, "two": 2, "three": 3, "four": 4, "five": 5,
+    "six": 6, "seven": 7, "eight": 8, "nine": 9, "ten": 10,
+    "eleven": 11, "twelve": 12, "thirteen": 13, "fourteen": 14,
+    "fifteen": 15, "sixteen": 16, "seventeen": 17, "eighteen": 18,
+    "nineteen": 19, "twenty": 20,
+}
 
-On each of these pages there is a "Locations" section 
+def _parse_quantity(text:str) -> int | None:
+    """
+    If the description starts with a clear count word ('One', 'Two', ...),
+    return that integer, else None (e.g., 'Over twenty', 'Sometimes', etc).
+    """
+    if not text:
+        return None
+    first = text.strip().lower().split()[0]
+    return _NUM_WORDS.get(first)
```

Comment:
This should be more generically named, and added to the general utilities, I can see it being useful elsewhere.